source/hardware/controllers/gemini_g4v.rst

```diff
@@ -144,7 +144,7 @@ Hot Cue Mode
 To enable Hot Cue mode, press the :hwlabel:`HOT CUE` button.
 When the performance pads are in Hot Cue mode, the Hot Cue button will be lit up.
 
-Pressing a performance pad when in Hot Cue mode will control the corresponding numbered Hot Cue (Performance Pad 1 controls Hot Cue 1, etc) on the virtual deck.
+Pressing a performance pad when in Hot Cue mode will control the corresponding numbered Hot Cue (:hwlabel:`Pad 1` controls Hot Cue 1, etc) on the virtual deck.
```

Comment:
This is Pad actually labeled as "Pad 1" on the controller? If not, don't use `:hwlabel:` here.